.github/workflows/release-please.yml

```diff
@@ -0,0 +1,15 @@
+name: Run Release Please
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
You don't actually have to specify that token. It will use that by default according to the docs.